### DISCUSSION TITLE
One Tagger Auto-Tag

### DISCUSSION THREAD

**Discussion by RalphRichards** [Marekkon5/onetagger#432] [Q&A]
I am attempting to auto-tag my music library but the m3u playlist file is not being recognized when i drag the file into the drag and drop section. Do i need to drag the playlist file for the tags to be recognized in rekordbox? what do I do after the analyze is complete?

**Comment by shytec** [comment]
I also can not work with the m3u playlikst file. It wont work. Its windows 10 latest and One tagger Latest. Because i also work with mp3tag (just looking and deleting some tags) and use Traktor Pro 3.8.11 its very handy when imcan use the m3u playlist funstion for better management. What can i do?
